src/components/breadcrumb/index.njk

```diff
@@ -11,7 +11,8 @@ A breadcrumb trail consists of a list of links to the parent pages of the curren
 }}
 
 ## When to use this component
-Use the breadcrumbs component when you need to help users understand and move between the multiple levels of a website.
+Use the breadcrumbs component when you need to help users understand and move between the multiple levels of a website. We recomend not using it on
+pages that are only one level deep in the page hierarchy. As we would like to avoid 'home' being the only link, when it’s directly under a menu that also has ‘home’.
```

Comment:
```suggestion
Use the breadcrumbs component when you need to help users understand and move between the multiple levels of a website.
```
I think we'll have to remove this comment about not displaying it when home is in the nav - it's a trade-off but I think the convention and consistency of always retaining the breadcrumb is more important than the lack of value in this small case